<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notes_editor</class>
 <widget class="QMainWindow" name="notes_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="notesEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_toggleSendControls"/>
   <addaction name="action_sendAll"/>
   <addaction name="action_sendLine"/>
   <addaction name="action_sendSelected"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_toggleSendControls">
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../mudlet.qrc">
     <normaloff>:/icons/arrow-right_grey-16x.png</normaloff>
     <normalon>:/icons/arrow-down_grey-16x.png</normalon>
     <activeoff>:/icons/arrow-right-16x.png</activeoff>
     <activeon>:/icons/arrow-down-16x.png</activeon>:/icons/arrow-right_grey-16x.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_sendAll">
   <property name="text">
    <string>Send All</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_sendLine">
   <property name="text">
    <string>Send Line</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_sendSelected">
   <property name="text">
    <string>Send Selected</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
